### ISSUE TITLE
[theme:install] should trigger [theme:download] if the theme is not available.

### ISSUE THREAD

**Issue by itsdarrylnorris** [hechoendrupal/drupal-console#2076]
### Problem / Motivation

An expected behavior from Drush user is if the user is going to install a project (module/theme) that is not currently available is going to try to download first and the installed it.

This have happened to me as well to other new DrupalConsole users. For instance - [Install module via Drupal Console fails](http://drupal.stackexchange.com/questions/186598/install-module-via-drupal-console-fails) some people are thinking that this is even a bug because is something expected already.
### Solution

We need to add the support for the `[theme:install]`.

**Comment by jmolivas** [comment]
:+1: We should implement this as we did on `module:install`

**Comment by danielrose28** [comment]
I'm going to give this one a try if that's ok? I'll let ya know when/if I have any questions.

**Comment by darol100** [comment]
@danielrose28, Feel free to started working on that.

**Comment by danielrose28** [comment]
So I've been trying to get this done when I have some free time and haven't forgotten about it. I was curious. In the english version of the  `theme.install.yml` & `theme.uninstall.yml` files, am I basically just re-writing the messages to advise to "download" themes that are missing?

I've started on this one. You'll see the `no-themes: 'You must provide or download themes to install.'` that I have started on. I've just re-worded a couple of the lines after that, but wasn't sure if I was actually doing it right.
![screen shot 2016-04-29 at 11 34 18 pm](https://cloud.githubusercontent.com/assets/12372245/14933642/8d7422a0-0e63-11e6-9163-babac968f340.png)
